src/pipeline/xhr/headers.js

```diff
@@ -0,0 +1,3 @@
+export const REQUEST_MARKER   = 'hammerhead|xhr|request-marker-header';
```

Comment:
Add warning comment, since this file used both by client and server.
Use `export default` like in other grouped constants files